sdk/appconfiguration/app-configuration/src/policies/throttlingRetryPolicy.ts

```diff
@@ -131,10 +131,8 @@ export class ThrottlingRetryPolicy extends BaseRequestPolicy {
         return await waitForTimeoutOrAbortOrResolve({
           timeoutMs: delayInMs,
           abortSignal: httpRequest.abortSignal,
-          actionFn: async () => {
-            return await this.sendRequest(httpRequest.clone());
-          },
-          timeoutMessage: `ServiceBusy: Unable to fulfill the request in ${delayInMs}ms when retried.`
+          actionFn: () => this.sendRequest(httpRequest.clone()),
```

Comment:
This is my fault, what you wanted (as you pointed out offline) is this one:

https://github.com/Azure/azure-sdk-for-js/blob/713d4335705c84a43df2964613cfdbefbc23451f/sdk/core/core-amqp/src/util/utils.ts#L179

So that way it's just:

```typescript
await delay(blah, abortSignal);
this._sendRequest()
```

Reply:
yup yup

Reply:
please review again!